app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,274 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'metrics_page', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'https://www.gstatic.com/charts/loader.js' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+    <%= render partial: 'practices/form/metrics', formats: [:js] %>
+  <% end %>
+<% end %>
+
+<%= render 'practice_editor_sidenav' %>
+<div class="grid-container position-relative">
+  <div class="grid-row grid-gap practice-editor-min-height">
+    <div class="desktop:grid-col-3 grid-col-auto z-bottom">&nbsp;</div>
+
+    <div id="metrics" class="desktop:grid-col-9 grid-col-12 p0">
+      <div>
+      <%= render partial: "shared/messages", locals: {small_text: false} %>
+    </div>
+      <section class="usa-section padding-top-0 p0-bottom" aria-label="Metrics">
+        <div class="grid-row flex-align-end">
```

Comment:
Let's add the class `margin-top-4` here to bring the `h1` more in line with the side nav. I know it's right based on the designs, but all of our other editor pages have a good bit of padding between the breadcrumbs and the `h1`.

Here's what I mean: 
![Screen Shot 2021-01-14 at 10 43 05 AM](https://user-images.githubusercontent.com/34111449/104613751-8811aa00-5655-11eb-992f-010734d822f9.png)

Reply:
I agree.